modules/configure-acs-kube-security-platform.adoc

```diff
@@ -21,34 +21,41 @@ annotations:
 
 [NOTE]
 ====
-An annotation can be on the deployment or the namespace.
+You can create an annotation for the deployment or the namespace.
 ====
 
-If you have configured the deployment or namespace with annotation, the {product-title-short} platform sends the alert to the email specified in the annotation. Otherwise, it sends the alert to the default recipient.
+If you have configured the deployment or namespace with an annotation, the {product-title-short} platform sends the alert to the email specified in the annotation. Otherwise, it sends the alert to the default recipient.
 
 .Procedure
 
 . Navigate to *Platform Configuration* -> *Integrations*.
 . Under the *Notifier Integrations* section, select *Email*.
-. Select the *New Integration* icon.
-. In the *Integration Name* field, enter a name for your email integration.
-. For *Email Server*, enter the address of your email server. The email server address is the fully qualified domain name and the port number. An example is `smtp.example.com:465`.
-. Enter a username and password of a service account that is used for authentication.
-. Optional: Specify the name of the sender, if it is different from the default value.
-. Specify the email address that will receive the notifications in the *Default Recipient* box, or enter an annotation key for *Annotation key for recipient*.
-. Optional: Turn on the *Disable TLS certificate validation (insecure)* toggle to send email without TLS. You should not disable TLS unless you are using StartTLS.
+. Select *New Integration*.
+. In the *Integration name* field, enter a name for your email integration.
+. In the *Email server* field, enter the address of your email server. The email server address includes fully qualified domain name (FQDN) and the port number. An example is `smtp.example.com:465`.
+. Optional: If you are using unauthenticated SMTP, select *Enable unauthenticated SMTP*. This is insecure and not recommended, but might be required for some integrations. For example, you might need to enable this option if you use an internal server for notifications that does not require authentication. 
++
+[NOTE]
+====
+You cannot change an existing email integration that uses authentication to enable unauthenticated SMTP. You must delete the existing integration and create a new one with *Enable unauthenticated SMTP* selected. 
+====
+. Enter the user name and password of a service account that is used for authentication.
+. Optional: Enter the name you want to appear in the FROM header of email notifications in the *From* field.
+. Optional: Specify the name you want to appear in the SENDER header of email notifications in the *Sender* field.
+. Specify the email address that will receive the notifications in the *Default recipient* box, or enter an annotation key in *Annotation key for recipient*.
```

Comment:
This should be split. Default recipient is always required.

Annotation is optional. If they provide it _and_ the deployment _or_ the namespace has a key with this value, then it will send it to that email. Otherwise it uses the Default recipient.

Therefore the default cannot be empty.

Reply:
I split this and then reiterated what an annotation does in case someone missed it in the first part of the module.